lib/Parse/ParseDecl.cpp

```diff
@@ -5694,9 +5694,7 @@ void Parser::parseAbstractFunctionBody(AbstractFunctionDecl *AFD) {
     if (!Body.hasCodeCompletion() && BS->getNumElements() == 1) {
       auto Element = BS->getElement(0);
       if (auto *stmt = Element.dyn_cast<Stmt *>()) {
-        auto kind = AFD->getKind();
-        if (kind == DeclKind::Var || kind == DeclKind::Subscript ||
-            kind == DeclKind::Func ) {
+        if (isa<AccessorDecl>(AFD) || isa<FuncDecl>(AFD)) {
```

Comment:
AccessorDecl is a subclass of FuncDecl so only the RHS of the `||` is needed.

Reply:
Fixed!